### PR TITLE
#5079 Fix metrics not correctly handle error issue

### PR DESCRIPTION
**Pull request by konekoya**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #5079)
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/store/epics/pipeline/updatePipelineMetricsEpic.js [hunk 1]
```diff
@@ -69,4 +60,13 @@ export default (action$) =>
         ),
       ),
     ),
+    catchError((err) => {
```
Comment (konekoya) [inline ohara-manager/client/src/store/epics/pipeline/updatePipelineMetricsEpic.js:63]: Ensure error is caught and shown for users

File: ohara-manager/client/src/components/Pipeline/Paper/Paper.js [hunk 2]
```diff
@@ -589,13 +589,17 @@ const Paper = React.forwardRef((props, ref) => {
...
-                elementView.toggleMetrics(false); // Stop displaying metrics once an error occurs
+                elementView.toggleMetrics(false);
               }
 
               break;
             case CELL_STATUS.stopped:
               elementView.enableMenu();
 
+              if (!isTopic) {
+                elementView.toggleMetrics(false);
```
Comment (konekoya) [inline ohara-manager/client/src/components/Pipeline/Paper/Paper.js:600]: Don't show loading metrics text for "stopped" connector

File: ohara-manager/client/src/store/epics/__tests__/pipeline/updatePipelineMetricsEpic.test.js [hunk 3]
```diff
@@ -16,22 +16,26 @@
...
 jest.mock('api/pipelineApi');
 
 const paperApi = {
   updateMetrics: jest.fn(),
 };
 
-beforeEach(jest.resetAllMocks);
+beforeEach(() => {
+  jest.restoreAllMocks();
+  jest.resetAllMocks();
```
Comment (saivirtue) [inline ohara-manager/client/src/store/epics/__tests__/pipeline/updatePipelineMetricsEpic.test.js:37]: `restoreAllMocks` will cover the logic of `resetAllMocks` did :)

> **mockFn.mockRestore()**
Does everything that mockFn.mockReset() does, and also restores the original (non-mocked) implementation.

Comment (konekoya) [reply ohara-manager/client/src/store/epics/__tests__/pipeline/updatePipelineMetricsEpic.test.js:37]: I was wondering about the same thing but they have to be used together or the test will fail...

**Comment by konekoya** [comment]
retry manager-e2e

File: ohara-manager/client/src/store/epics/__tests__/pipeline/updatePipelineMetricsEpic.test.js (hunk 3 above)
Comment (saivirtue) [reply ohara-manager/client/src/store/epics/__tests__/pipeline/updatePipelineMetricsEpic.test.js:37]: It's OK :)
We could dig it in the future 😄

**Pull request merged**